### DISCUSSION TITLE
What paper repository asreview use?

### DISCUSSION THREAD

**Discussion by italojsv** [asreview/asreview#745] [Q&A] [answered]
I'm a scientometric student and I'm think to use asrewview in my master degree work(article recommendation system) but I would like to know where from the papers is extracted.

**Comment by SagevdBrand** [comment, marked as answer]
Hi there @italojsv  😊 

Awesome that you're considering ASReview for you master's degree work! For a systematic review, the papers, articles and such, are retrieved from search engines such as Scopus, PubMed, Web of Science, and so on.  Perhaps your library has a list of search engines per discipline, or a list of search engines in general. 

In this [blogpost](https://asreview.nl/blog/asreview-class-101/) the process of using ASReview in a systematic review is explained, hopefully you will find it useful!

All the best and good luck!
